k2/torch/bin/online_decode.cu

```diff
@@ -110,7 +114,7 @@ int main(int argc, char *argv[]) {
     ./bin/online_decode \
       --use_ctc_decoding true \
       --jit_pt <path to exported torch script pt file> \
-      --tokens <path to tokens.txt> \
+      --bpe_model <path to pretrained BPE model> \
```

Comment:
We don't support passing `bpe.model` anymore as we have removed `sentencepiece`.

Reply:
Yes, I saw it. thanks!